calculator/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Button from "./Components/Button";
+import Result from "./Components/Result";
+import "./App.css";
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = { result: "" };
+  }
+  //right column stuff
+  onClick = (symbol) => {
+    if (symbol === "=") {
+      this.solve();
+    } else if (symbol === "AC") {
+      this.clear();
+    } else if (symbol === "sign") {
+      this.switchSign();
+    } else {
+      this.setState({ result: (this.state.result + symbol).toLocaleString("en") });
+    }
+  };
+  //all clear
+  clear = () => {
+    this.setState({
+      result: "",
```

Comment:
I think it would be better to keep track of operator in the state than it would just have result and use eval. Eval has properties within it that do things like automatically round up